Bio/SeqIO/InsdcIO.py

```diff
@@ -650,11 +647,21 @@ def _get_date(record):
             "NOV",
             "DEC",
         ]
+        if isinstance(date, datetime) or isinstance(date, datetime_date):
+            date = f"{date.day:02d}-{months[date.month - 1]}-{date.year}"
```

Comment:
You can check both classes at once with `isinstance(date, (datetime, datetime_date))` - see https://docs.python.org/3/library/functions.html#isinstance

However, we only need to check the `date` class as `datetime` is a subclass of `date` -- see https://docs.python.org/3/library/datetime.html#available-types or:

```pycon
>>> import datetime
>>> d = datetime.date(2025,6,30)
>>> d
datetime.date(2025, 6, 30)
>>> isinstance(d, datetime.date)
True
>>> isinstance(d, datetime.datetime)
False
>>> dt = datetime.datetime(2025, 6, 30)
>>> dt
datetime.datetime(2025, 6, 30, 0, 0)
>>> isinstance(dt, datetime.date)
True
>>> isinstance(dt, datetime.datetime)
True
```

Reply:
Thanks, I wan't aware of the isinstance with a tuple of class. Addressed